client/src/selectors/index.js

```diff
@@ -353,3 +353,16 @@ export const addNeighboursLimitSelector = (state: State): number | null =>
 export const isItCandidatesListSelector = (state: State, props: ContextRouter): boolean =>
   props.location.search.includes('kandidati_bratislava') ||
   props.location.search.includes('kandidati_prezident')
+
+export const autocompleteSuggestionEidsSelector = (state: State, query: string): Array<number> => (
+  state.entitySearch[query] && state.entitySearch[query].eids
+) || []
+
+export const autocompleteSuggestionsSelector =
+  (state: State, value: string): Array<string> => {
+    const details = state.entityDetails
+    const eids = (state.entitySearch[value] && state.entitySearch[value].eids) || []
```

Comment:
Is there a reason for this duplicate code? (Could use `allEntityDetailsSelector` for details and `autocompleteSuggestionEidsSelector` for eids)

Reply:
Nope, no reason. Going to use mentioned functions.